src/tests/jf_test_1.f90

```diff
@@ -430,9 +430,10 @@ subroutine print_json_variable(json,p,finished)
 
     end subroutine print_json_variable
 
-end module jf_test_1_mod
+    end module jf_test_1_mod
```

Comment:
Why the extra indentation? This should be removed AFAICT

Reply:
If it's replacing tab with spaces, it's still not a great practice to introduce trivial whitespace changes into projects with your patches, but easy to see how it could have gone unnoticed.